vpt/capabilities/initiatable.py

```diff
@@ -0,0 +1,16 @@
+'''The base mixin for the nodes that can be started and stopped.'''
+from abc import ABC, abstractmethod
+
+
+class Initiatable(ABC):
+    '''Base class for nodes that can be started and stopped.
+       Should be used for sources and processors.'''
+    stopped: bool
+
+    @abstractmethod
+    def start(self):
+        '''Start the node's data stream.'''
+
+    @abstractmethod
+    def stop(self):
+        '''Start the node's data stream.'''
```

Comment:
How about the default implementations of `stopped = True` and `stopped = False` for cases when they suffice?

Reply:
They never do

Reply:
They do in most of the sources. This is already the implementation their